extension/src/python/index.ts

```diff
@@ -2,7 +2,21 @@ import { join } from 'path'
 import { getVenvBinPath } from './path'
 import { getProcessPlatform } from '../env'
 import { exists } from '../fileSystem'
-import { createProcessWithOutput } from '../processExecution'
+import { Logger } from '../common/logger'
+import { createProcess, Process, ProcessOptions } from '../processExecution'
+
+const sendOutput = (process: Process) =>
+  process.all?.on('data', chunk =>
+    Logger.log(chunk.toString().replace(/(\r?\n)/g, ''))
+  )
+
+export const createProcessWithOutput = (options: ProcessOptions) => {
+  const process = createProcess(options)
+
+  sendOutput(process)
+
+  return process
+}
```

Comment:
Are we allowed to use this to do things like [getting the ast from the project's Python files](https://docs.python.org/3/library/ast.html#command-line-usage) ?

Reply:
You could definitely try. Take a look at https://github.com/microsoft/vscode-python/tree/main/src/client/languageServer to see what those guys do (outside of the closed source language server)

Reply:
Why is that closed, btw?

Reply:
💰